kmod/src/scoutfs_trace.h

```diff
@@ -2020,9 +2020,9 @@ DEFINE_EVENT(scoutfs_quorum_message_class, scoutfs_quorum_recv_message,
 
 TRACE_EVENT(scoutfs_quorum_loop,
 	TP_PROTO(struct super_block *sb, int role, u64 term, int vote_for,
-		 unsigned long vote_bits, struct timespec64 timeout),
+		 unsigned long vote_bits, time64_t timeout_sec, long timeout_nsec),
```

Comment:
It's not worth the noise to get the trace to have secs/nsecs.  Just have it take a u64 nsecs and have the trace event caller use ktime_to_ns().